File: eAirFlow.Model/Models/Notification.cs
using System;
using System.Collections.Generic;

namespace eAirFlow.Model.Models
{

    public class Notification
    {
        public int NotificationId { get; set; }

        public int? UserId { get; set; }

        public string? Message { get; set; }

        public DateTime? SentAt { get; set; }
        public User? User { get; set; }
        public bool IsSeen { get; set; }


    }
}

File: UI/eairflow_desktop/lib/models/notification.dart
class NotificationModel {
  int? notificationId;
  int? userId;
  String? message;
  String? sentAt;
  bool? isSeen;

  NotificationModel({
    this.notificationId,
    this.userId,
    this.message,
    this.sentAt,
    this.isSeen,
  });

  factory NotificationModel.fromJson(Map<String, dynamic> json) {
    return NotificationModel(
      notificationId: json["notificationId"],
      userId: json["userId"],
      message: json["message"],
      sentAt: json["sentAt"],
      isSeen: json["isSeen"],
    );
  }
}
